knexfile.ts

```diff
@@ -21,3 +20,6 @@ module.exports = {
   },
   useNullAsDefault: false,
 };
+
+export default knexConfig;
+module.exports = knexConfig;
```

Comment:
Might be a dumb question. But why module.exports and export default?

Reply:
I literally do not know but it doesn’t work otherwise